x/mock/simulation/simulate.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 // AppStateFn returns the app state json bytes
-type AppStateFn func(r *rand.Rand, accs []Account, genesisTimestamp time.Time) json.RawMessage
+type AppStateFn func(r *rand.Rand, accs []Account, genesisTimestamp time.Time) (json.RawMessage, []Account, string)
```

Comment:
Update the comment above `AppStateFn`. Also I'd suggest that output variable names are used in this type definition, specifically to make clear what this `string` represents

Reply:
++

Reply:
updated, added variable names